src/components/product.js

```diff
@@ -461,7 +477,7 @@ export default class Product extends Component {
    * @return {String}
    */
   get onlineStoreURL() {
-    const identifier = this.handle ? this.handle : this.id;
+    const identifier = this.handle ? this.handle : atob(this.id).split('/').pop();
```

Comment:
This is probably an  anti-pattern. We should likely add the online store URL as a field on product. The domain is not always going to be the myshopify domain (which is what's configured here).